packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -340,6 +357,10 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
       titleId,
       descriptionId,
       isNavExpandable,
+      hasDrawer,
+      isDrawerExpanded,
+      onDrawerToggle,
+      drawerToggleButtonText,
```

Comment:
Should we have a default for the text.  Otherwise maybe we should not render the button at all if not text is provided.  When the there is not text, the button is rendered but you can't click it since it has not width. 
 @mcoker What do you think?

Reply:
I like this idea @tlabaj! Setting "Open Drawer" as the default for `drawerToggleButtonText`. I can add "defaults to Open Drawer" in the prop description, and the the consumer would know to set that prop on `Wizard` if they want some other custom text

Reply:
On second thought, it might be a better idea to have this text customizable at every step! Will push with a new update in a moment...

Reply:
@tlabaj hmm... looking at this again, we updated the open drawer functionality to be a demo. The user should be able to use any kind of button, anywhere they want to toggle the drawer. If I'm reading the example code correctly, I don't think we want to include that button in the component itself - this should be defined in the component example (or demo example). https://github.com/patternfly/patternfly-react/pull/7323/files#diff-038302fe0ddb5ac4ec5b59ca6bf16bd2ef6fa81fcf8bb56572a40d78df73b59eR111-R120

Also as a side note, the `.pf-u-*` classes used will not work if users are not importing the utility classes themselves, since those don't ship with patternfly by default. They would need to import https://www.patternfly.org/v4/utilities/float and https://www.patternfly.org/v4/utilities/spacing

Reply:
So the wizard is not composable. I think the best we can do on the react side is pass a button rather than the text for the button we create internally.  That would also take care of the utility class issue, since they will not be part of the component.

Reply:
@jenny-s51 It looks like the implementation would work for the scenario where the drawer toggle button is in the wizard content.  What is not accounted for here is the second scenario in the [design](https://marvelapp.com/prototype/9389af6/screen/86742796) where the button is inline as part of a paragraph in the content. 
Core also has this [demo](https://pf4.patternfly.org/components/wizard/html-demos#with-drawer-and-informational-step) with the inline variation

Reply:
@tlabaj thank you! Updated to include an inline drawer toggle demo in modal and in page!